core/org.osate.ui/src/org/osate/ui/navigator/AadlProjectDependenciesActionProvider.java

```diff
@@ -0,0 +1,55 @@
+package org.osate.ui.navigator;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.IMenuManager;
+import org.eclipse.ui.dialogs.PreferencesUtil;
+import org.eclipse.ui.navigator.CommonActionProvider;
+import org.eclipse.ui.navigator.ICommonMenuConstants;
+
+public class AadlProjectDependenciesActionProvider extends CommonActionProvider {
+	private static final String ID = "org.eclipse.ui.propertypages.project.reference";
+	private static final String[] ID_LIST = { ID };
+	private static final Object DUMMY_DATA = new Object();
+
+	private final Action editDependenciesAction;
+
+	public AadlProjectDependenciesActionProvider() {
+		editDependenciesAction = new Action() {
+			{
+				setText("Edit Project Dependencies...");
+				setToolTipText("Open the dependencies project property page.");
```

Comment:
change to "Open the project references property page"

Reply:
done